content/blog/2019-01-03-ozunconf18.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: "ozunconf18"
+title: Continuing to Grow Community Together at ozunconf, 2018
+authors:
+  - name: Nicholas Tierney
+    url: https://www.njtierney.com
+    twitter: nj_tierney
+# twitterImg: img/blog-images/2019-01-03-ozunconf18/fun-photo.jpg
+date: 2019-01-03
+categories: blog
+topicid: 925
+tags:
+- r
```

Comment:
for tags add
- unconf
- ozunconf
- ozunconf18

delete "925" from topicid and I will add topicid at publication time

Reply:
Done!